test/e2e/apimachinery/aggregator.go

```diff
@@ -460,8 +491,31 @@ func TestSampleAPIServer(f *framework.Framework, image string) {
 	if len(unstructuredList.Items) != 0 {
 		framework.Failf("failed to get back the correct deleted flunders list %v from the dynamic client", unstructuredList)
 	}
+}
 
-	cleanTest(client, aggrclient, namespace)
+func testDiscovery(f *framework.Framework) {
+	client := f.ClientSet
+	resources, err := client.Discovery().ServerResourcesForGroupVersion("wardle.k8s.io/v1alpha1")
+	if err != nil {
+		framework.Failf("failed to find the discovery doc for %v: %v", resources, err)
+	}
+	found := false
+	// The calculation of the hash value is for test purpose only. Clients
+	// shouldn't rely on the exact hash value, which is an implementation
```

Comment:
But they should rely on it only changing at particular times, no?

Reply:
Yes.